content/_includes/oss.macros.njk

```diff
@@ -56,7 +56,7 @@ category: file
       {% endif %}
     {% endcall %}
     {{ post.subtitle(item.data.sub, collections) }}
-    {{ post.byline(item, collections, dateformat='since') }}
+    {{ post.byline(item, collections, dateformat='short') }}
```

Comment:
@mirisuzanne I couldn't find any record of a `since` format (also looked in the repo history), but I thought `short` made some sense here.

Reply:
@jgerigmeyer I already fixed this in the other branch. I think 'range' was the proper value.